### ISSUE TITLE
problem with yubikey errors

### ISSUE THREAD

**Issue by mts19** [keepassxreboot/keepassxc#2059]
## Expected Behavior
To use yubikey second slot static password to unlock the database. Windows 7 and Solus.
## Current Behavior
When the yubikey is plugged in and I start keepassxc I get a massive amount of unhandled errors.
This happens in the journal on start of keepassxc.
## Possible Solution
At the moment I have had to remove the yubikey and am pasting the password from a note.

## Steps to Reproduce (for bugs)
Start the application when the yubikey is plugged in.
Also happens when locking the database.
To then enter the password with the yubikey press I have to remove it and re-insert it.
On the first press prior to re-insertion nothing auto-types.

## Context
These entries are totally spamming the journal.
Previously I was using keepass2 with mono and had no problems with the yubikey for inserting the static passsword. (I use the second slot in the yubikey for the static password.)
There are many more repeated entries but have just shown some.

`Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0/0003:1050:0010.000C
Jun 19 20:46:07 solus budgie-wm[943]: g_array_unref: assertion 'array' failed
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'bind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'bind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'bind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'bind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'bind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'bind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'bind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'bind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'bind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0
Jun 19 20:46:07 solus upowerd[716]: unhandled action 'unbind' on /sys/devices/pci0000:00/0000:00:1d.0/usb4/4-1/4-1.2/4-1.2.3/4-1.2.3:1.0`

## Debug Info
KeePassXC - Version 2.3.3
Revision: 0a155d8
Distribution: AppImage

Libraries:
- Qt 5.9.4
- libgcrypt 1.8.1

Operating system: Solus 3.9999
CPU architecture: x86_64
Kernel: linux 4.16.15-76.current

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by droidmonkey** [comment]
Are you sure this is a KeePassXC issue? Do you get these errors when the application is not running? Does KeePass with Mono still work without error?

**Comment by mts19** [comment]
@droidmonkey 
I have already uninstalled keepass2 and mono.
The yubikey worked without problems.
Also there is no problem in Windows.

I decided to try an older kernel (lts) and all the warnings have gone.
I still have a slight problem.
When locking the database upon pressing the yubikey to unlock it turns the yubikey off.
I then have to remove and re-insert the yubikey to enter the static password.
Bit of a pain but it works without all the errors.

Now using Linux solus 4.9.107-97.lts rather than the 4.16.15-76 kernel.
